modules/nf-core/bwa/mem/main.nf

```diff
@@ -24,6 +28,9 @@ process BWA_MEM {
     def args2 = task.ext.args2 ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     def samtools_command = sort_bam ? 'sort' : 'view'
+    def extension = args2.contains("--output-fmt sam") ? "sam" :
+                    args2.contains("--output-fmt cram") ? "cram" :
```

Comment:
Maybe also check for the short form? `-O`

Reply:
Done!